rskj-core/src/test/java/co/rsk/peg/whitelist/WhitelistSupportImplTest.java

```diff
@@ -262,6 +264,33 @@ void setLockWhitelistDisableBlockDelay_whenDisableBlockDelayPlusBtcBlockchainBes
         assertEquals(WhitelistResponseCode.DISABLE_BLOCK_DELAY_INVALID.getCode(), actualResult);
     }
 
+    @Test
+    void verifyLockSenderIsWhitelisted_whenOneOffLockWhitelistAddressIsWhitelisted_shouldReturnTrue() {
+        Transaction tx = TransactionUtils.getTransactionFromCaller(signatureCache, WhitelistCaller.AUTHORIZED.getRskAddress());
+        whitelistSupport.addOneOffLockWhitelistAddress(tx, btcAddress.toString(), BigInteger.TEN);
+
+        boolean actualResult = whitelistSupport.verifyLockSenderIsWhitelisted(btcAddress, Coin.SATOSHI, 0);
+
+        assertTrue(actualResult);
```

Comment:
I think we should also test that the address is consumed after the verification

Reply:
Done!

Reply:
What about in the other test?

Reply:
Unlimited whitelisting means that the entries are never fully consumed so there's nothing to do here.

Reply:
Cool, thanks for the clarification